src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/WebserviceController.php

```diff
@@ -409,4 +396,47 @@ private function getErrorMessages()
             DuplicateWebserviceKeyException::class => $this->trans('This key already exists.', 'Admin.Advparameters.Notification'),
         ];
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return array<string, bool|string|null>
+     */
+    private function getWebServiceStatus(Request $request): array
+    {
+        $webserviceConfiguration = $this->get('prestashop.admin.webservice.form_data_provider')->getData();
+        $webserviceStatus = [
+            'isEnabled' => (bool) $webserviceConfiguration['enable_webservice'],
+            'isFunctional' => (bool) $webserviceConfiguration['enable_webservice'],
```

Comment:
```suggestion
            'isFunctional' => false,
```

Should be false by default, no need to perform assertion, if you do it just after when the service is enabled